.travis.yml

```diff
@@ -2,15 +2,22 @@ language: python
 python:
     - 2.7
     - 2.6
+addons:
+    apt:
+        packages:
+            - oracle-java8-set-default
 install:
     - source utils/init-environment
     - pip install codecov
-services:
-    - elasticsearch
-# This is required for elasticsearch
+
+# Sleep is required for elasticsearch.
 before_script:
     - sleep 10
-script: coverage run --source shelf/ -m unittest discover -p *_test.py tests -b
+    - curl -v http://localhost:9200/
```

Comment:
It doesn't really slow down the test much either so I don't see a reason to remove it.